lib/abstract/tsc.js

```diff
@@ -2,14 +2,13 @@ module.exports = function (option, done) {
   'use strict';
   var async         = require('async');
   var fs            = require('fs');
-  var util          = require('util');
   var child_process = require('child_process');
   var dirname       = require('../config/dirname');
 
   async.waterfall([function (callback) {
     var exists        = fs.existsSync(dirname.current);
     if (!exists) {
-      util.print('tvm use <version>');
+      console.log(console.log('tvm use <version>'));
```

Comment:
WTF? why nested `console.log`calls? 💣

Reply:
I overlooked it! Will fix it. Patch is also welcomed ;)